.github/workflows/django.yml

```diff
@@ -75,6 +75,7 @@ jobs:
           PGHOSTADDR: 127.0.0.1
         run: |
           python manage.py collectstatic > /dev/null
+          mkdir -p media
```

Comment:
media folder also required for CI as it doesn't run make install as it does not need `migrate` command